packages/terra-toggle-section-header/src/ToggleSectionHeader.jsx

```diff
@@ -101,6 +113,7 @@ class ToggleSectionHeader extends React.Component {
       isTransparent,
       onOpen,
       onClose,
+      isOpen,
       ...customProps
     } = this.props;
```

Comment:
The logic for the uncontrolled workflow needs to be suppressed when when `this.props.isOpen` is defined. Currently, the onClick action invokes a setState to update the open/close state of the component regardless of uncontrolled/controlled. Set state shouldn't be called when this component is controlled. 

https://github.com/cerner/terra-core/blob/4d631da3d237f68bfd1a70bf5b33562c4b6126e9/packages/terra-toggle-section-header/src/ToggleSectionHeader.jsx#L90-L92

Reply:
Fixed ed43374